.github/workflows/dependency-review.yml

```
@@ -7,9 +7,6 @@ jobs:
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout code
-        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
-      - name: Dependency Review
-        uses: actions/dependency-review-action@3b139cfc5fae8b618d3eae3675e383bb1769c019 # v4.5.0
-        with:
-          config-file: it-at-m/.github/workflow-configs/dependency_review.yaml@main
\ No newline at end of file
+      - id: build_docs
+        uses: it-at-m/lhm_actions/action-templates/actions/action-dependency-review@hupling-patch-1
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)